deploy/build.xml

```diff
@@ -19,10 +19,8 @@
 	<property name="serlio.version.minor" value="2" />
 	<property name="serlio.version.patch" value="0" />
 	<property name="serlio.version.mmp" value="${serlio.version.major}.${serlio.version.minor}.${serlio.version.patch}" />
-
-	<condition property="serlio.version.mmpPre" value="${serlio.version.major}.${serlio.version.minor}.${serlio.version.patch}-${serlio.version.preRelease}" else="${serlio.version.major}.${serlio.version.minor}.${serlio.version.patch}">
-		<isset property="maya2018.preRelease" />
-	</condition>
+	<property name="serlio.version.preRelease" value="dev.0" />
```

Comment:
Move the dash from `serlio.version.mmpPre` to `serlio.version.preRelease` otherwise we have a dash at the end of the release version (where preRelease is empty)

Reply:
makes sense, will fix that asap